templates/CRM/Contribute/Form/Selector.tpl

```diff
@@ -26,6 +26,10 @@
 {include file="CRM/common/pager.tpl" location="top"}
 
 {strip}
+  <div class="description">
+    {ts}Click arrow to view payment details.{/ts}
```

Comment:
I am not a native English speaker so I might be wrong, but isn't it more correct to be : 

```
Click on the arrow to view the payment details.
```

Reply:
@omarabuhussein @eileenmcnaughton I'm not sure that this text actually adds any value to the UI and might be better removed altogether?

Reply:
ohh - I shy away totally at UI decisions. Gramatically I think no 'the' is probably better than with the 'the' - but I lean heavily on people like Andrew Hunt, Stoob, Pete D & Lcdweb for UI decisions - I didn't ping them because I thought pinging all 4 might be overkill :-)

Reply:
I meant removing the entire description :-) Thoughts @agh1 @lcdservices ?

Reply:
according to @lcdservices 

> And at this point, the arrows are ubiquitous throughout the system, so it may not need explanation.

I think I agree, so I think removing the entire description make sense here.